src/data/content-items/data-source.js

```diff
@@ -414,9 +415,75 @@ export default class ContentItem extends oldContentItem.dataSource {
     return this.coreSummaryMethod(root);
   };
 
-  getSermonNotes = async ({ value: guid }) => {
+  getSermonNoteComments = async (contentID) => {
+    const { Auth } = this.context.dataSources;
+    const { primaryAliasId } = await Auth.getCurrentPerson();
+    const comments = await this.request('Notes')
+      .filter(
+        `CreatedByPersonAliasId eq ${primaryAliasId} and EntityId eq ${contentID} and NoteTypeId eq ${
+          ROCK_MAPPINGS.SAVED_SERMON_NOTE_TYPE_ID
+        }`
+      )
+      .get();
+
+    const commentsHash = {};
+    comments.forEach(({ id, text: data }) => {
+      const { apollosParentID, text } = JSON.parse(data);
+      commentsHash[apollosParentID] = {
+        id: createGlobalId(id, 'Note'),
+        text,
+      };
+    });
+    return commentsHash;
+  };
+
+  saveSermonNoteComment = async (contentID, parentID, text) => {
+    const { Auth } = this.context.dataSources;
+    const { primaryAliasId } = await Auth.getCurrentPerson();
+
+    // find a saved note for the parent
+    const contentNotes = await this.request('Notes')
+      .filter(`CreatedByPersonAliasId eq ${primaryAliasId}`)
+      .andFilter(`EntityId eq ${parseGlobalId(contentID).id}`)
+      .andFilter(`NoteTypeId eq ${ROCK_MAPPINGS.SAVED_SERMON_NOTE_TYPE_ID}`)
+      .get();
```

Comment:
These lines are basically the same as lines 419-427. Probably could combine this into a function and just call it from both places. No need to duplicate.

Reply:
Also, a consistent naming structure. In one place it's `comments` and in the other it's `contentNotes`.

Reply:
The same for consistent general structure. I don't think I care much if you do `this and that and this` or `this.andFilter(that).andFilter(this)` but we should probably be consistent whichever we do. Particularly in the same set of changes or related items.

Reply:
addressed these